docs: refine development document (#2)
chaos-mesh/website: docs/development_guides/setup_env.md

3. Make sure [Docker](https://docs.docker.com/install/) is installed and running on your local machine.

> And you need to run docker registry, more detail refer to [Docker Regitry](https://docs.docker.com/registry/). Then set environment variable `DOCKER_REGISTRY` with the registry address:

Reviewer: Is this an extra step? please be clear - don't just add instructions randomly following existing steps.

Author: I add a new step for it

> 4. Make sure Docker Registry is running, more detail refer to [Docker Regitry](https://docs.docker.com/registry/). Then set environment variable `DOCKER_REGISTRY` with the registry address:

Reviewer: ```suggestion
4. Make sure [Docker Regitry](https://docs.docker.com/registry/) is running. Set the environment variable `DOCKER_REGISTRY` with the registry address:
```

Author: done

hack/kind-cluster-build.sh
```

> The above script will create a Kubernetes cluster by kind. When you don't need this cluster, you can run the following command to delete it:

Reviewer: ```suggestion
The above script creates a Kubernetes cluster via Kind. When you are done with it, run the following command to delete it:
```

Author: done
